.github/workflows/deploy.yml

```
@@ -0,0 +1,25 @@
+name: deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build
+        run: |
+          npm ci
+          npm run-script build
```

Interesting - I'd never heard of `npm ci` before, and assumed it would run tests (CI meaning "continuous integration") but it actually runs a ["clean install"](https://docs.npmjs.com/cli/v8/commands/npm-ci). Nice.

Have suggested naming the install and build steps separately, for clarity and faster debugging if one of the steps fails.

```suggestion
      - name: Install
        run: npm ci
      - name: Build
        run: npm build
```

---

Ah, I thought `build` was special like `start` and didn't need a `run` or `run-script` prefix!